### DISCUSSION TITLE
Lexical error on line 1: Unrecognized text. in v4

### DISCUSSION THREAD

**Discussion by simonmaass** [tailwindlabs/tailwindcss#14994] [Help]
**What version of Tailwind CSS are you using?**

For example: v4.0.0-alpha.30

**What build tool (or framework if it abstracts the build tool) are you using?**

"@tailwindcss/node@4.0.0-alpha.30"
"@tailwindcss/postcss@4.0.0-alpha.30"
"@tailwindcss/vite@4.0.0-alpha.30"

**What version of Node.js are you using?**

v20.11.1

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**Describe your issue**

 WARN  [vite:css] Lexical error on line 1: Unrecognized text.                                                                                                                                     17:04:20

  Erroneous area:
1: infinity * 1px
^..^
1117|    }
1118|    .rounded-full {
1119|      border-radius: calc(infinity * 1px);
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
1120|    }
1121|    .rounded-lg { (x2)

![image](https://github.com/user-attachments/assets/baca0ffb-066b-4109-b52a-274669d690ca)

**Comment by philipp-spiess** [comment]
Hey there! I'm as surprised as you to see this as being marked as unrecognized because it does work in browsers 😅 

Do you have some info on how to reproduce this? I've tried to run it inside a `Vite` playground (using the `@tailwindcss/vite` package) with both postcss and lightningcss as the configured preprocessor and I never saw this warning (neither in dev mode nor in production). 

Would be cool if we can track down where the parsing goes wrong and send a fix upstream.

**Reply by simonmaass** [reply]
This is my setup:

- nuxt@3.14.1592
- vite@5.4.11
- @tailwindcss/vite@4.0.0-alpha.34
- @nuxt/ui@3.0.0-alpha.9

**Reply by simonmaass** [reply]
@philipp-spiess it looks like quite some peope have this error

**Comment by benfeather** [comment]
I'm also seeing this error in a nuxt 3 project. My versions are:

nuxt: 3.14.1592,
tailwindcss: 4.0.0-beta.7
@tailwindcss/vite: 4.0.0-beta.7
vite: 5.4.11

**Reply by simonmaass** [reply]
@benfeather are u using nuxt/ui v3?

**Reply by benfeather** [reply]
@simonmaass No, i'm not using any UI library.

**Comment by ckvv** [comment]
I think it was a problem with the vite css processor, the problem was solved when I switched from postcss to lightningcss

**Comment by sk31park** [comment]
Has anyone solved this issue?

**Comment by eudj1n** [comment]
Same error on latest (not alpha) versions

**Comment by tired-engineer** [comment]
Same here with cssnano.

**Reply by ricky-harrys** [reply]
Removing cssnano fixed the errors on my builds

```json
"@remix-run/react": "^2.11.1",
"@tailwindcss/postcss": "^4.0.0",
"@tailwindcss/vite": "^4.0.3",
```

**Comment by eduayme** [comment]
Same error here with `nuxt 3.15.2` and `nuxt ui 3.0.0-alpha.10`

**Reply by simonmaass** [reply]
i also have this with nuxt 3.15.4 and nuxt/ui 3.0.0-alpha.12

**Comment by mshornikov** [comment]
Same issue with `nuxt 3.15.4`, `@tailwindcss/vite 4.0.1`, and `tailwindcss 4.0.1`

**Comment by jlemonz** [comment]
same man this upgrade to v4

**Comment by al-dubois** [comment]
same error here
@nuxt/ui 3.0.0-alpha.12 
nuxt 3.15.4 
tailwindcss 4.0.1

**Comment by zxdream64230** [comment]
same error here
@nuxt/ui 3.0.0-alpha.12

Isn't it a problem with nuxtui 3 since we all have the same problem ?

Have you noticed problems with padding in production vs localhost ?

I loose my vertiacal paddings and don't know if it's related to those errors ...

**Reply by al-dubois** [reply]
> same error here
> 
> @nuxt/ui 3.0.0-alpha.12
> 
> 
> 
> Isn't it a problem with nuxtui 3 since we all have the same problem ?
> 
> 
> 
> Have you noticed problems with padding in production vs localhost ?
> 
> 
> 
> I loose my vertiacal paddings and don't know if it's related to those errors ...

Yes I noticed the loss of vertical padding as well. Only when deploying, not in local dev. 

It looks like it's NuxtUI3 focused but I've seen multiple threads pointing fingers at TailwindCSS v4..

**Comment by terion-name** [comment]
same here with nuxt, still

**Reply by wongjn** [reply]
Consider providing a git repo that reproduces the issue and we can take a look for you.

**Reply by terion-name** [reply]
sorry, it was vite and plugins bugs, updated and tuned configs and now works
